src/app/(pages)/home/_components/OurValuesBlock.tsx

```diff
@@ -78,7 +78,8 @@ function Value({ data }: { data: ValueDataType }) {
       <div
         className={cn(
           "hidden md:flex top-1/2 w-[100px]  md:w-[180px] lg:w-[220px] aspect-square transform absolute -translate-y-1/2  justify-center items-center z-4 right-2 translate-x-full",
```

Comment:
we can also adjust this lg:w-[220px] to lg:w-[260px] and i think that would look big enough